tox.ini

```diff
@@ -15,3 +16,8 @@ commands = multilint
 
 [testenv:py35-codestyle]
 commands = multilint
+
+[testenv:py35-numpy]
+deps = -r{toxinidir}/requirements.txt
+    numpy
```

Comment:
If we add numpy like this, we enforce the users of the plugin to install NumPy, which is not desired IMO.